sdk/src/wallet/operations/transaction/mod.rs

```diff
@@ -90,7 +90,10 @@ where
         let block_id = match self
             .submit_basic_block(
                 Some(Payload::from(signed_transaction_data.payload.clone())),
-                options.as_ref().and_then(|options| options.issuer_id),
+                options
+                    .as_ref()
+                    .and_then(|options| options.issuer_id)
```

Comment:
Do you think option one should be preferred?

Reply:
Wasn't sure about this, but probably would be a bit better to prefer the transaction data one, since this was used when the tx with the allotments was created, will change

Reply:
Changed